src/lib/__tests__/uri.test.js

```diff
@@ -53,21 +66,32 @@ describe('uri', () => {
         address: 'U123456',
         params: {
           label: 'Rick Sanchez'
-        }
+        },
+        crypto: 'ADM',
+        protocol: "adm"
       })
     })
   })
 
   describe('generateURI', () => {
+    beforeAll(() => {
+      delete window.location
+      window.location = new URL('https://msg.adamant.im')
+    })
+
+    afterAll(() => {
+      delete window.location;
+    })
+
     it('without contactName', () => {
-      expect(generateURI('U123456')).toBe('adm:U123456')
-      expect(generateURI('U123456', '')).toBe('adm:U123456')
-      expect(generateURI('U123456', undefined)).toBe('adm:U123456')
+      expect(generateURI(Cryptos.ADM, 'U123456')).toBe('https://msg.adamant.im?address=U123456')
```

Comment:
I think we need to create a constant for base url `https://msg.adamant.im`

Reply:
Normally the hostname should be stored in environment variables. And depending on the environment, it can be e.g. `testing.adamant.im` for testing environment or `msg.adamant.im` for production. It will be fixed but not as part of this PR.

For now I'll create a constant in the test files to get rid of some repetitions.